apps/condo/domains/billing/schema/BillingIntegrationOrganizationContext.js

```diff
@@ -85,7 +89,43 @@ const BillingIntegrationOrganizationContext = new GQLListSchema('BillingIntegrat
             isRequired: false,
             type: Text,
         },
+
+        paymentsAllowedFrom: {
+            schemaDoc: 'Datetime from which receipts from this billing are able to be paid for',
+            type: DateTimeUtc,
+            isRequired: false,
+            access: {
+                create: access.canManagePaymentsRelatedFields,
+                update: access.canManagePaymentsRelatedFields,
+                read: true,
+            },
+        },
+
+        paymentsAllowedTo: {
+            schemaDoc: 'Datetime to which receipts from this billing are able to be paid for',
+            type: DateTimeUtc,
+            isRequired: false,
+            access: {
+                create: access.canManagePaymentsRelatedFields,
+                update: access.canManagePaymentsRelatedFields,
+                read: true,
+            },
+        },
+
+        isPaymentsAllowed: {
```

Comment:
And why this fields is stored inside context? Isn't it a part of any receipt? Like how much time I have for each receipt before it becomes non-actual

Reply:
Might do it a bit later